packages/gestalt/src/Pog.css

```diff
@@ -29,12 +31,12 @@
 .transparent.focused,
 .white.hovered,
 .white.focused {
-  background-color: rgba(0, 0, 0, 0.06);
+  background-color: var(--gestalt-colorTransparentGray60);
 }
 
 .transparent.active,
 .white.active {
-  background-color: rgba(0, 0, 0, 0.1);
+  background-color: var(--gestalt-colorTransparentGray100);
```

Comment:
Be sure to add this css car to this file too. Not sure if you'll get an error at build time but it's required for the legacy css file built for IE11 and currently used in prod